### PR TITLE
Enhanced filters Features/9076

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#9076](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9076)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Filters update

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### UI Changes

YES

### Browsers Tested

* [x] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

image
image
image
image

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+    private toggleMobileView() {
+        this.isVisible = !this.isVisible;
+    }
+
+    private clearFilters(): void {
+        this.selectedFilters = [];
+    }
+
+    private onExternalFilterSelection(filterName: string) {
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue:258]: as this method updates the selectedFilters, can we call this updateSelectedFilter?

**Review by leobcgov — CHANGES_REQUESTED** [review]
nice one, just one naming item to review

File: Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue (hunk 1 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue:258]: the catch is that this is only called when from an external call via the bus. IMHO updateSelectedFilter gives the impression that this updates it in a general way

Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue:258]: Mbe an item for retro to define our naming convention as for me method should have a meaningful name and its name should tell what it does. Approve this for now then :-)

**Review by leobcgov — APPROVED** [review]

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**